vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    gid: 9002
+    state: present
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /opt/minio
```

Comment:
If it needs a user home at all, why not /home/minio

Reply:
We don't have a {{ root_folder }}/home concept, so the normal /home is better.